### PR TITLE
feat: add the alby logo to the QR used for invoices

### LINKED ISSUE #2757: [Feature] Add logo to the QR codes for invoices
### Feature description

Adding the Alby logo to the QR code used for invoices

### Describe the solution

Currently we are using the `react-qr-code` package for creating QR code with invoice instead we can use the `react-qrcode-logo` package to add a logo to the QR codes.

### Describe alternatives

_No response_

### Additional context

image

### Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by Rithvik-padma**
### Describe the changes you have made in this PR

Added the Alby logo to the QR code used for invoices.

### Link this PR to an issue [optional]

Fixes #2757 

### Type of change

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes [optional]

Tk60gUEYmV

### How has this been tested?

Tested by creating an invoice.

### Checklist

- [x] Self-review of changed code
- [x] Manual testing

### CODE REVIEW

**Comment by Rithvik-padma** [comment]
@rolznz Can you please review the changes.
Thanks!

File: src/app/screens/ReceiveInvoice/index.tsx [hunk 1]
```diff
@@ -143,7 +143,14 @@ function ReceiveInvoice() {
     return (
       <>
         <div className="mt-4 relative p-8 bg-white rounded-lg shadow-sm ring-1 ring-black ring-opacity-5 flex justify-center items-center overflow-hidden">
-          <QRCode value={invoice.paymentRequest.toUpperCase()} />
+          <div className="relative flex flex-grid w-fit h-fit">
```
Comment (rolznz) [inline src/app/screens/ReceiveInvoice/index.tsx:146]: I don't think it needs to be a grid, there's only one element. I think if you change it to `flex justify-center items-center` you might also be able to remove the transform and translate classes on the img tag too.

Comment (Rithvik-padma) [reply src/app/screens/ReceiveInvoice/index.tsx]: Sure, I have made the changes
Thanks!

File: src/app/screens/ReceiveInvoice/index.tsx [hunk 2]
```diff
@@ -142,8 +142,15 @@ function ReceiveInvoice() {
     if (!invoice) return null;
     return (
       <>
-        <div className="mt-4 relative p-8 bg-white rounded-lg shadow-sm ring-1 ring-black ring-opacity-5 flex justify-center items-center overflow-hidden">
-          <QRCode value={invoice.paymentRequest.toUpperCase()} />
+        <div className="mt-4 relative p-8 bg-white dark:bg-surface-01dp rounded-lg shadow-sm ring-1 ring-black ring-opacity-5 flex justify-center items-center overflow-hidden">
+          <div className="relative flex items-center justify-center">
+            <QRCode value={invoice.paymentRequest.toUpperCase()} />
```
Comment (rolznz) [inline src/app/screens/ReceiveInvoice/index.tsx:147]: I think we missed this, it also shows the Alby QR code for Alby accounts. There it should show the Alby user's avatar if they have one.

See src/app/screens/Receive/index.tsx for the code that is used to display the Alby user avatar.

I see if the Alby user does not have an `avatarUrl` it'll use the randomly generated avatar. I think showing the Alby icon would be better here. @reneaaron @stackingsaunter what do you think?

Comment (Rithvik-padma) [reply src/app/screens/ReceiveInvoice/index.tsx:147]: Okay, then I will make changes for the first one.

**Comment by Rithvik-padma** [comment]
@rolnz I have the made the change for using the avatar if it's an Alby user or else the logo would be used.
I did not make any change to the static QR in the receive screen for now.
Thanks!

File: src/app/screens/ReceiveInvoice/index.tsx (hunk 2 above)
Comment (rolznz) [reply src/app/screens/ReceiveInvoice/index.tsx:151]: It looks good to me, apart from a bit of duplicated code. But maybe that can be cleaned up later. Thanks @Rithvik-padma !

**Review by rolznz — APPROVED** [review]
tACK

**Comment by pavanjoshi914** [comment]
update this pr and tested it with breeze, phoenix, WOS, bluewallet,strike, normal camera scanner

**Review by bumi — APPROVED** [review]
utACK.

thanks! \o/

**Pull request merged**